### ISSUE TITLE
Packages unavailable (404)

### ISSUE THREAD

**Issue by HitCoder9768** [enigma-dev/ExtraPackages#4]
It appears that everything here following Alure (so zlib and onwards) is causing a dropbox 404. This means that the Linux installer cannot progress beyond alure.

**Comment by RobertBColton** [comment]
Can you show a screenshot of that output? I think it has done this for a few years now, and it's not a problem because it's just the MacOS depends which fail. So although the script should be updated, I believe it's relatively harmless.

**Comment by HitCoder9768** [comment]
https://i.imgur.com/Z5d4JAm.png this is a screenshot of the output.

I've tested about 5 of the dropbox links in the md5 file (as stated, zlib and onwards) and all of them fail.

**Comment by HitCoder9768** [comment]
I've created a pastebin, but I'm still fairly confident it's the urls.
https://pastebin.com/BdJrD6JP

when I do `java -jar lateralgm.jar` it doesn't show compile options.

**Comment by HitCoder9768** [comment]
also in terms of the 404 (it occurred to me that was the "output" you wanted to see) I've checked the urls in my browser, this is the 404 output of zlib.epackage
https://i.imgur.com/ZN3hiaY.png

**Comment by RobertBColton** [comment]
Ah, yeah, just like you said, it's failing to find zlib, which ENIGMA needs to compress your resources onto the exe.

https://github.com/enigma-dev/ExtraPackages/blob/master/Packages.md5
So basically the script works by crawling that file I just linked ``Packages.md5`` and comparing their md5 checksum to see if a new version should be downloaded.

I would have to ask @JoshDreamland what was in that dropbox because I am not sure. I know we tell you to install zlib from the shell using your package manager, so I don't know what else would need to be installed or why it's not just including with ENIGMA.

**Comment by RobertBColton** [comment]
Also, as you can see I have not updated that repository since "Sep 14, 2016"

I'll forward this to Josh.

**Comment by HitCoder9768** [comment]
I've already installed most of those dependencies through my package manager, but that seems to make no visible change.

Okay, thanks for the help so far.

**Comment by RobertBColton** [comment]
Yes, this repo here I created so that I could actually update those links and maintain them better. This is actually a fork of the original https://github.com/enigma-dev/Enigma-packages/blob/master/packages.md5 and we can see the link is 404'ing there too. Josh will have an answer.

**Comment by RobertBColton** [comment]
@HitCoder9768 actually, I am going to delete this repository and merge it into the original because we don't need it anymore. Can you please refile this issue on https://github.com/enigma-dev/Enigma-packages/issues

**Comment by HitCoder9768** [comment]
should be done
